lumibot/tools/yahoo_helper.py

```diff
@@ -236,7 +224,7 @@ def fetch_symbol_info(symbol, caching=True, last_needed_datetime=None):
 
         # Caching is disabled or no previous data found
         # or data found not up to date
-        data = YahooHelper.download_symbol_info(symbol)
+        data = YahooHelper.get_symbol_dict(symbol)
```

Comment:
Because of the changes to the underlying get_symbol_dict() function, this no longer caches any meaningful data.